pnpm-lock.yaml

```diff
@@ -9662,7 +9662,7 @@ packages:
 
   puppeteer@19.11.1:
     resolution: {integrity: sha512-39olGaX2djYUdhaQQHDZ0T0GwEp+5f9UB9HmEP0qHfdQHIq0xGQZuAZ5TLnJIc/88SrPLpEflPC+xUqOTv3c5g==}
-    deprecated: < 22.8.2 is no longer supported
+    deprecated: < 22.6.4 is no longer supported
```

Comment:
Was this intended?

Reply:
just `pnpm i` after mergin / installing dependencies. did not review pnpm lock. does this cause issues?